.github/workflows/tests.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI test
+
+# Trigger the workflow on push and cron
+on:
+  push:
+  schedule:
+  - cron:  '5 0 * * *'
+
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: goanpeca/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        channels: conda-forge
+    - name: Install prerequisites
+      shell: bash -l {0}
+      run: |
+        conda install pocl islpy pyopencl pytest pip
+        pip install git+https://gitlab.tiker.net/inducer/{loopy,meshmode,dagrt,leap,modepy,meshpy,grudge}
+        pip install -e .
+    - name: Run tests
+      shell: bash -l {0}
+      run: |
+        pytest
+
+  MacOS:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: goanpeca/setup-miniconda@v1
+      with:
+        auto-update-conda: true
+        channels: conda-forge
```

Comment:
I imagine that this will also source packages from the conda `defaults` channel, which we do not want. Maybe adding `nodefaults` might do the trick.

x-ref: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html